srcpkgs/mpdscribble/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'mpdscribble'
 pkgname=mpdscribble
-version=0.22
-revision=6
-build_style=gnu-configure
+version=0.23
+revision=1
+build_style=meson
 hostmakedepends="pkg-config"
-makedepends="libmpdclient-devel glib-devel libcurl-devel"
+makedepends="libmpdclient-devel libcurl-devel boost-devel libgcrypt-devel"
 short_desc="MPD audio scrobbler (e.g. last.fm)"
 maintainer="Georg Schabel <gescha@posteo.de>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/MusicPlayerDaemon/mpdscribble"
-distfiles="http://www.musicpd.org/download/${pkgname}/${version}/${pkgname}-${version}.tar.bz2"
-checksum=1cde2b5f8c70f0e3e6f059d01bf0b0f2eac2b29fbcd14b01a8a5103e603feb41
+changelog="https://raw.githubusercontent.com/MusicPlayerDaemon/mpdscribble/v0.23/NEWS"
```

Comment:
Either use the `master` branch (preferable) or `v${version}`

Reply:
Done, thanx!